assets/sass/components/key-metrics/_googlesitekit-km-widget-tile.scss

```diff
@@ -113,5 +113,12 @@
 				font-weight: $fw-medium;
 			}
 		}
+
+		.googlesitekit-km-widget-tile__table-plain-text {
```

Comment:
Very nitpicky, but could we implement the 0.2px letter spacing here?

Reply:
Updated ✔️